packages/compass-shell/src/modules/worker-runtime.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+import { createRequire } from 'module';
+
+const { WorkerRuntime } = (() => {
+  // Workaround for webpack require that overrides global require
+  const req = createRequire(path.resolve(__filename));
```

Comment:
```suggestion
  const req = createRequire(__filename);
```

because `__filename` is always an absolute path? unless this is also something that ends up being hijacked by webpack

Reply:
Ah, yeah, thanks! Yeah, webpack does hijack it by default, but I disabled it with `node: false` so it definitely should not be necessary anymore

Reply:
Done in a420520